charts/lab-partner/values.yaml

```diff
@@ -8,8 +8,9 @@ mongodb:
   service:
     name: lab-partner-mongodb
 
-image:
-  name: lab-partner
+image: liatrio/lab-partner
```

Comment:
The value names here make the chart more confusing. Please change these back to `image.name` and `image.pullPolicy` not `image` and `imageConfig.pullPolicy` and update the references in the skaffold.yaml and templates.